paddle/fluid/operators/math/blas_impl.h

```diff
@@ -627,6 +662,67 @@ void Blas<DeviceContext>::MatMul(const framework::Tensor &mat_a,
         dim_a.stride_, dim_b.stride_);
   }
 }
+
+#if defined(PADDLE_WITH_MKLML) && !defined(PADDLE_WITH_CUDA)
```

Comment:
This `#if` is still inconsistent with `#ifdef` for `MatMulWithHead()` declaration.

Reply:
there is already a `#ifdef PADDLE_WITH_MKLML` outside the definition of `MatMulWithHead`, like below:
```C
#ifdef PADDLE_WITH_MKLML
...

#if !defined(PADDLE_WITH_CUDA)
  template <typename T>
  void MatMulWithHead(const framework::Tensor& mat_a,
                      const MatDescriptor& dim_a,
                      const framework::Tensor& mat_b,
                      const MatDescriptor& dim_b, T alpha, int head_number,
                      framework::Tensor* mat_out, T beta) const;
#endif
#endif
```
so they are consistent with other code:)

Reply:
OK